CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 - In MAUI Android apps, generate and inject UUID to APK and upload ProGuard mapping to Sentry with the UUID ([#4532](https://github.com/getsentry/sentry-dotnet/pull/4532))
 - Fixed WASM0001 warning when building Blazor WebAssembly projects ([#4519](https://github.com/getsentry/sentry-dotnet/pull/4519))
 
+### Experimental
+
+- Remove `ExperimentalAttribute` from all _Structured Logs_ APIs, and remove `Experimental` property from `SentrySdk`, but keep `Experimental` property on `SentryOptions` ([#4567](https://github.com/getsentry/sentry-dotnet/pull/4567))
+
```

Comment:
question: I am totally uncertain what section to put this change in.

It's technically a source and binary breaking change, but to (previously) Experimental-marked APIs.
Doesn't seem like `Features` nor `Fixes`.
I'm not sure if instead a `BREAKING CHANGES` section is more suited ... but uncertain if that fits with Experimental changes too 🤔

Reply:
I totally missed that ... changed to `### API Changes` for consistency ... thanks!